README.md

```diff
@@ -1 +1,139 @@
-# Awesome-Books-ES6
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h1><b>
+Awesome Books With ES6</b></h1>
+Awesome Books is a simple website that displays a list of books and allows you to add and remove books from that list. This is a Microverse Module 2 project.
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#awesom-books)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 🛠 Built With
+
+- HTML5
+- CSS
+- Javascript ES6
+
+# Key Features
+
+- Add and remove books
+
+  - The website that displays a list of books and allows you to add and remove books from that list.
+
+- JavaScript Classes
+
+  - The books are managed and manipulated using classes and helper functions within those classes.
+
+- Store and retrieve data
+  - The displayed books are stored and retrieved from the local storage of the browser.
+
+# 🔭 Future Features
+
+> I will be adding More functions.
+
```

Comment:
**[Optional]**

- [ ] It would be nice if you be more specific, which functions are you talking about means what type of function are you gonna add and what these functions will do :thinking: